packages/salesforcedx-vscode-core/test/vscode-integration/conflict/timestampConflictDetector.test.ts

```diff
@@ -273,6 +275,7 @@ describe('Timestamp Conflict Detector Execution', () => {
     expect(differStub.callCount).to.equal(0);
     expect(results.different).to.eql(new Set<TimestampFileProperties>());
   });
+  */
 
   it('Should not report differences if the files match', async () => {
```

Comment:
@jeffb-sfdc was this test working with the old code? I'm wondering if there are any failing tests we could add with the old code so that we could remove the commented test above, but still capture the use case that you've fixed. Maybe something like 'should not report differences if the files match BUT timestamps are different'?